savant/client/__init__.py

```diff
@@ -3,6 +3,12 @@
 from .builder.sink import SinkBuilder
 from .builder.source import SourceBuilder
 from .frame_source import FrameSource
-from .frame_source.jpeg import JpegSource
 from .log_provider import LogProvider
 from .log_provider.jaeger import JaegerLogProvider
+
+try:
+    from .jpeg_source import JpegSource
+except ImportError:
+    import logging
+
+    logging.warning('Failed to import JpegSource. Install OpenCV to enable it.')
```

Comment:
Why root logging and not logger object with 'insight.savant' prefix?

Reply:
Changed the logger.